plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/DeployPreferencesTest.java

```diff
@@ -16,60 +16,151 @@
 
 package com.google.cloud.tools.eclipse.appengine.deploy;
 
+import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.text.IsEmptyString.isEmptyString;
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
-import com.google.cloud.tools.eclipse.test.util.project.TestProjectCreator;
 import org.eclipse.core.resources.IProject;
-import org.junit.Rule;
+import org.junit.Before;
 import org.junit.Test;
 
 public class DeployPreferencesTest {
```

Comment:
The other thing that would be useful here is to test that (1) the `IEclipsePreferences` haven't changed, and (2) they change after a `save()`.